src/app/machines/views/Machines.test.tsx

```diff
@@ -229,7 +230,13 @@ describe("Machines", () => {
         lists: {
           "78910": machineList,
         },
-        filters: [machineFilterGroupFactory()],
+        filters: [
+          machineFilterGroupFactory({
+            key: FilterGroupKey.Workloads,
+            loaded: true,
+            options: [{ key: "animal:springbox", label: "animal: springbok" }],
```

Comment:
TIL about springbok. Appreciate the effort you put into choosing labels in tests 😉 

```suggestion
            options: [{ key: "animal:springbok", label: "animal: springbok" }],
```

Reply:
Thanks for catching my spelling mistakes :joy: